Sources/Commands/SwiftBuildTool.swift

```diff
@@ -145,56 +57,20 @@ public class SwiftBuildTool: SwiftTool<BuildToolMode, BuildToolOptions> {
         }
     }
 
-    override class func usage(_ print: (String) -> Void = { print($0) }) {
-        //     .........10.........20.........30.........40.........50.........60.........70..
-        print("OVERVIEW: Build sources into binary products")
-        print("")
-        print("USAGE: swift build [mode] [options]")
-        print("")
-        print("MODES:")
-        print("  -c, --configuration <value>   Build with configuration (debug|release) [default: debug]")
-        print("  --clean [<mode>]              Delete artifacts (build|dist) [default: build]")
-        print("")
-        print("OPTIONS:")
-        print("  -C, --chdir <path>       Change working directory before any other operation")
-        print("  --build-path <path>      Specify build/cache directory [default: ./.build]")
-        print("  --color <mode>           Specify color mode (auto|always|never) [default: auto]")
-        print("  -v, --verbose            Increase verbosity of informational output")
-        print("  -Xcc <flag>              Pass flag through to all C compiler invocations")
-        print("  -Xlinker <flag>          Pass flag through to all linker invocations")
-        print("  -Xswiftc <flag>          Pass flag through to all Swift compiler invocations")
-        print("")
-        print("NOTE: Use `swift package` to perform other functions on packages")
-    }
-    
-    override class func parse(commandLineArguments args: [String]) throws -> (BuildToolMode, BuildToolOptions) {
-        let (mode, flags): (BuildToolMode?, [BuildToolFlag]) = try Basic.parseOptions(arguments: args)
-    
-        let options = BuildToolOptions()
-        for flag in flags {
-            switch flag {
-            case .chdir(let path):
-                options.chdir = path
-            case .verbose(let amount):
-                options.verbosity += amount
-            case .xcc(let value):
-                options.flags.cCompilerFlags.append(value)
-            case .xld(let value):
-                options.flags.linkerFlags.append(value)
-            case .xswiftc(let value):
-                options.flags.swiftCompilerFlags.append(value)
-            case .buildPath(let path):
-                options.buildPath = path
-            case .enableNewResolver:
-                options.enableNewResolver = true
-            case .buildTests:
-                options.buildTests = true
-            case .colorMode(let mode):
-                options.colorMode = mode
-            }
-        }
-    
-        return try (mode ?? .build(.debug, UserToolchain()), options)
+    override class func bindArguments(parser: ArgumentParser, binder: ArgumentBinder<BuildToolOptions>) {
```

Comment:
I would call this `defineArguments` not bind. It does both, but it seems like the most important thing it does is define them.